borg/archiver.py

```diff
@@ -1458,6 +1465,10 @@ def setup_signal_handlers():  # pragma: no cover
 
 
 def main():  # pragma: no cover
+    # provide borgfs behaviour when the main script/executable is named borgfs
+    if os.path.basename(sys.argv[0]) == "borgfs" and (len(sys.argv) < 2 or sys.argv[1] != "mount"):
```

Comment:
With main_mount() gone again, we indeed do not (works both for entry_point scripts and the binary). Thanks! (And I should be off to bed...)